sdk/monitor/test-resources.json

```diff
@@ -92,6 +93,16 @@
         "principalId": "[parameters('testApplicationOid')]",
         "scope": "[resourceGroup().id]"
       }
+    },
+    {
+      "type": "Microsoft.Insights/components",
```

Comment:
Is this going to create an Application Insights resource when you execute the tests?